kubernetes/celestia-local/scripts/start-bridge.sh

```diff
@@ -11,10 +11,24 @@ else
   echo "genesis hash received: $genesis_hash"
 fi
 
+# NOTE - this is a hack to get the token to the token-server.
+# busybox's httpd doesn't support url rewriting, so to make
+# the ingress rule path `/` we write the token to index.html
+TOKEN=$(celestia bridge auth admin \
+  --node.store "$home_dir/bridge" \
+  --keyring.accname validator)
+mkdir -p "$home_dir"/token-server
+echo "$TOKEN" > "$home_dir"/token-server/index.html
+
 export CELESTIA_CUSTOM="test:$genesis_hash"
   # --p2p.network "test:$celestia_custom"
 export GOLOG_LOG_LEVEL="debug"
-exec ./celestia bridge start \
+
+exec celestia bridge start \
   --node.store "$home_dir/bridge" \
+  --core.ip 127.0.0.1 \
+  --core.rpc.port "$celestia_app_host_port" \
   --gateway \
```

Comment:
We should probably add a `--rpc.port` flag for setting the json-rpc port and a variable in configmap